### PR TITLE
Mailpoet 4360 remove old model usage from rendering MAILPOET-4360

### PR DESCRIPTION
**Pull request by samnajian**
This PR removes old Newsletter model support from rendering 

# How to test:
1) 
- Create a newsletter 
- View the preview and verify it's rendered correctly.
- Send it to your email and verify it's rendered correctly (in MailHog or your inbox)
2) 
- Adjust a transactional email
- Invoke the email sending
- Verify the sent email is rendered correctly 

[MAILPOET-4360]

[MAILPOET-4360]: https://mailpoet.atlassian.net/browse/MAILPOET-4360?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/tests/docker/docker-compose.yml [hunk 1]
```diff
@@ -104,6 +104,8 @@ services:
       MYSQL_PASSWORD: wordpress
     volumes:
       - /dev/shm:/dev/shm
+    ports:
```
Comment (rodrigoprimo) [inline mailpoet/tests/docker/docker-compose.yml]: Could you please elaborate on why this is needed and why in this PR? Asking as it is not mentioned anywhere in the PR description or the Jira ticket.

**Review by rodrigoprimo — COMMENTED** [review]
Code changes look good to me and I could confirm that rendering the emails still works in all the scenarios.

I just left one comment related to a change that I'm not sure why is needed and why it was included in this PR.

File: mailpoet/tests/docker/docker-compose.yml (hunk 1 above)
Comment (samnajian) [reply mailpoet/tests/docker/docker-compose.yml]: The test db was never exposed to the dev machine making the debugging a hassle, added this but then thought it makes sense to have in trunk as well for future debugging

Comment (samnajian) [reply mailpoet/tests/docker/docker-compose.yml]: FYI: commit removed, it'll be added under another ticket/PR

**Review by rodrigoprimo — APPROVED** [review]

**Review by veljkho — APPROVED** [review]

**Pull request merged**